packages/terra-compact-interactive-list/src/utils/keyHandlerUtils.js

```diff
@@ -0,0 +1,310 @@
+const focusableElementSelector = 'a[href]:not([tabindex=\'-1\']), area[href]:not([tabindex=\'-1\']), input:not([disabled]):not([tabindex=\'-1\']), '
+        + "select:not([disabled]):not([tabindex='-1']), textarea:not([disabled]):not([tabindex='-1']), button:not([disabled]):not([tabindex='-1']), "
+        + "iframe:not([tabindex='-1']), [tabindex]:not([tabindex='-1']), [contentEditable=true]:not([tabindex='-1'])";
+
+/* NOTE: This method doesn't include check for element.closest('[inert]') === null in purpose
+ * The method is caled on initial render to ensure the cell has no interactable elements, and set the cell tabindex based on that.
+ * on render one of the parent elements has inert="" which made it always return no interactible elements for every cell.
+*/
+export const getFocusableElements = (parentElement) => [...parentElement.querySelectorAll(`${focusableElementSelector}`)].filter(
+  element => !element.hasAttribute('disabled')
+      && !element.getAttribute('aria-hidden')
+      && window.getComputedStyle(element).visibility !== 'hidden',
+);
```

Comment:
We have a similar method in focusManagement.js in the Table component.  I would think we would not want to duplicate it all.  Same comment for the other methods.

Reply:
You should not need to change it.  You just need to update your jest tests appropriately.  If you check the other jest tests, there is code to handle providing an offset value.  The logic should work for your component.

Reply:
@cm9361 huge thank you for the help with tests to accommodate the full original logic from terra-table. I fixed the tests and currently added getFocusableElements is the original one that is implemented in terra-table.

In order to reuse the same logic rather than copy-paste it, I added the export for the `getFocusableElements` method from the `terra-table`. Once the terra-table is released, I will remove copied getFocusableElements method from the compact interactive list and attach the one exported from the terra-table. Would that work?
https://github.com/cerner/terra-framework/pull/1919/commits/cdbf56a32d01b0831f45994d9fef08cdce5c9d12

Reply:
You can use a relative path instead of having to add a package dependency.  That logic was released in terra-table quite a while ago though, if I am not mistaken.

Reply:
Correct, imported from terra-table: https://github.com/cerner/terra-framework/pull/1919/commits/e7343db5f21961d67aaf2ef57d4d2577e7bb1ec4